### DISCUSSION TITLE
Generating migrations

### DISCUSSION THREAD

**Discussion by callumagmo** [adrien2p/medusa-extender#43] [Q&A] [answered]
I was trying to generate migrations instead of writing migration queries on my own.
https://github.com/typeorm/typeorm/blob/master/docs/migrations.md#generating-migrations

After several tries, it does not generate any queries inside the Up/Down methods. I believe is due to the ormconfig.json in the medusa package is pointing to their own directory, which cannot read migrations outside the folder. (Let me know if im wrong)
https://github.com/medusajs/medusa/blob/master/packages/medusa/ormconfig.json

Any suggestion on this?

Besides, notice any new migrations will be auto applied during startup, any way to run it manually instead?
https://github.com/adrien2p/medusa-extender/blob/main/src/loaders/migrations.loader.ts

**Comment by adrien2p** [comment, marked as answer]
You can check the CLI doc [here](https://adrien2p.github.io/medusa-extender/#/?id=cli-medex)

**Reply by adrien2p** [reply]
i hope that it helps you, let me know if you have any other questions. Ill be glad to help you out
